panels/dev-service/ha-panel-dev-service.html

```diff
@@ -49,7 +49,7 @@
       }
 
       .attributes tr:nth-child(even) {
-        background-color: #eee;
+        background-color: var(--paper-listbox-background-color,#eee)
```

Comment:
Ok, changed this now. So we also now need to define `--table-row-alternative-background-color` and `--table-row-background-color` in ha-style.html, right?